src/main/java/sirius/db/jdbc/schema/BasicDatabaseDialect.java

```diff
@@ -146,6 +147,12 @@ public String generateDropTable(Table table) {
         return MessageFormat.format("DROP TABLE `{0}` ", table.getName());
     }
 
+    @Nullable
+    @Override
+    public String generateRenameTable(Table table) {
+        return MessageFormat.format("ALTER TABLE `{0}` RENAME `{1}`", table.getOldName(), table.getName());
```

Comment:
```suggestion
        return MessageFormat.format("ALTER TABLE `{0}` RENAME TO `{1}`", table.getOldName(), table.getName());
```

Reply:
or am I totally wrong?

Reply:
https://stackoverflow.com/q/886786/2448440 scheint total datenbank spezifisch zu sein. Speziell bei Mariadb ist das "to" wohl möglich, aber nicht nötig: https://mariadb.com/kb/en/alter-table/

Reply:
Let's keep TO in the default and permit dialects to change this if necessarry.